rfc/2022/2022-08-23_CIT_hostnames.md

```diff
@@ -0,0 +1,246 @@
+  - Once implemented, the new Apigee gateway will continue to honor requests to `api.va.gov/mobile/*` for older versions of the Mobile Flagship app.
+  - Once implemented, the new Apigee gateway will continue to honor requests to `api.va.gov/vanotify` and route them to the VANotify service until that team agrees to retire the endpoint.
+  - Kong Team will need to update routing to vets-api as it moves into EKS environment in different environments.
+
+
+#### **Infrastructure Team**
+  - Removes api.va.gov references from the revProxy. This should only happen once Apigee has been successfully set up as a new API gateway and fully owns the api.va.gov domains.
+  - While the revProxy still handles the api.va.gov domain, ensures that new rules added to the `nginx_api_server.conf` file get replicated in `ngix_new_api_server.conf`.
+  - Prevents new references to api.va.gov to be added to the vets-api config files `*-settings.local.yml.j2` and their EKS counterparts. 
+  - **Note:** At this time only the Dev website is in EKS. As other environments migrate to EKS, routing configurations will need to be updated. The Kong team will need to be alerted of the move to EKS, so that they can update their own routing from Kong to vets-api.
+
+    Reference: [Enabling new *platform-api.va.gov domains in EKS](https://vfs.atlassian.net/wiki/spaces/CLOUD/pages/2263351375).
+  - Schedule annual TLS/SSL cert renewals for all platform-api subdomains.
+
+
+#### **Documentation**
+  - The Platform Content Team coordinates with contributing teams to edit documentation to use the *platform-api.va.gov hostname.
+  - The Console UI Team coordinates with contributing teams to edit documentation to use the *platform-api.va.gov hostname.
+
+
+## **Risks**
+  - **Lack of parity in environments and deployment processes:** Consideration should be given to completing the migration of all environments to EKS prior to rolling out *platform-api.va.gov.
+    - Reduces complexity in Release Plan processes.
+    - Reduces complexity in Post-Release issue resolution.
+  - **Staging downtime:** A window of time will be required to allow teams to test and mitigate any unforeseen issues. Authentication flows and day-to-day development tasks may be interrupted.
+  - **Production downtime:** There may be a short period of downtime or instability in authentication flows.
+  - **Unidentified vets-api clients:** While we believe that the VA.gov website and the mobile app are the only clients of vets-api, any unidentified client would encounter 404 errors after the Apigee migration. To ward against that possibility, the Infrastructure Team could conduct a thorough examination of the revproxy logs to identify client IP addresses involved in vets-api requests that did not originate from either VA.gov or the mobile app.
+  - **Unidentified issues:** Issues that are not identified in the Staging release may affect Production.
+
+
+## **Appendix**
```

Comment:
Reminder to add "Hostname Transition Story Map" from CIT Confluence. Be aware that Confluence items may not be directly viewable by non-Platform teams.

Reply:
Added the Story Map to the Appendix. We're going to change this file to Excel format and put it in a more universally accessible location along with the other operational docs.